### ISSUE TITLE
Document fields in the JSON report

### ISSUE THREAD

**Issue by dclong** [pandas-profiling/pandas-profiling#637]
**Missing functionality**

The JSON report has lots of fields but unfortunately, they are very confusing and hard to figure out their meanings. For example, I thought `is_unique` tells whether a column has unique value or not, but it doesn't seem to be the case. Also, I have no idea what `n_unique` means. Based on the name, I'd guess it is then number of unique values, but it doesn't seem to be the case either. 

**Proposed feature**

Document fields in the JSON report. 

**Alternatives considered**

**Additional context**

**Comment by sbrugman** [comment]
Agreed, it would be a great addition to have a JSON schema. Contributions welcome.

To answer your question: `n_unique` is the number of unique values (i.e. occur exactly once). You might be looking for `n_distinct`, which is the number of distinct values.

**Comment by dclong** [comment]
Thank you, @sbrugman! I confused myself on `unique` vs `distinct`.
